### PR TITLE
Update PcdcPrivacyNotice.md

### PR DESCRIPTION
**Pull request by wandersen01**
Edited version

### CODE REVIEW

File: docs/PcdcPrivacyNotice.md [hunk 1]
```diff
@@ -1,128 +1,132 @@
-![Logo](img/PcdcLogo.png)
-# PRIVACY NOTICE
+![Logo](Images/PcdcLogo.png)
```
Comment (grugna) [inline docs/PcdcPrivacyNotice.md:1]: Same, Image folder

File: docs/PcdcPrivacyNotice.md [hunk 2]
```diff
@@ -1,128 +1,132 @@
...
+    <li>What are your data protection rights?</li>
+    <li>What are cookies?</li>
+    <li>How do we use cookies?</li>
+    <li>What types of cookies do we use?</li>
+    <li>How to manage your cookies</li>
+    <li>Privacy policies of other websites</li>
+    <li>Changes to our privacy policy</li>
+    <li>How to contact us</li>
+    <li>Remedies and right to file a complaint</li>
+ </ul>
```
Comment (grugna) [inline docs/PcdcPrivacyNotice.md:25]: I would keep the same format for all the bullet points, is there a reason why this is in HTML vs the others are using Markdown language?

Comment (wandersen01) [reply docs/PcdcPrivacyNotice.md:25]: I wrote this one first and I already knew some HTML. Once I read more on Markdown I figured out it's easier for most things so I switched for later documents. I can put it into Markdown quick.

Comment (wandersen01) [reply docs/PcdcPrivacyNotice.md]: I switched the unordered lists to markdown but kept some of the headings in HTML so they can be centered.

**Pull request closed without merging**